docs/setup-react-native.md

```diff
@@ -81,3 +81,25 @@ try {
   console.log(error)
 }
 ```
+
+## Using BBS Signatures
+
+When using AFJ inside the React Native environment, temporarily, a dependency for creating keys, sigining and verifying
+with bbs keys must be swapped. Inside your package.json the following must be added:
+
+```diff
++   "resolutions": {
++     "@mattrglobal/bbs-signatures": "react-native-bbs-signatures@0.1.1",
```

Comment:
should be `@animo-id/react-native-bbs-signatures` with version `0.0.3` (we probably want to release 0.1.0 right?)

Reply:
Yes when I finished some work on it I will release a 0.1.0